snapcraft/internal/common.py

```diff
@@ -269,6 +269,7 @@ def get_os_release_info():
     with open("/etc/os-release") as os_release_file:
         os_release_dict = {}
         for line in os_release_file:
-            key, value = line.rstrip().split('=')
-            os_release_dict[key] = value.strip('"')
+            entry = line.rstrip().split('=')
+            if len(entry) == 2:
+                os_release_dict[entry[0]] = entry[1].strip('"')
```

Comment:
Would you mind adding a unit test to `snapcraft/tests/test_common.py` with your failing os-release and one that doesn't have an empty line?